net-p2p/ipfs-desktop-bin/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>dawn_xu_helloworld@163.com</email>
```

Comment:
Please use the email address you registered with bugs.gentoo.org.

https://wiki.gentoo.org/wiki/Project:GURU#The_regulations
> 7. Users that list themselves as maintainers must use an e-mail address known to bugs.gentoo.org in metadata.xml and keep it up to date in case it changes.

Reply:
This is a alias name for my email registered in bugs.gentoo.org. But I won't leave my phone numbers on Internet, I wold change my gentoo bugzilla's email.

Reply:
Unfortunately, Bugzilla wouldn't know whether it's an alias or not.
```
$ curl 'https://bugs.gentoo.org/rest/user?names=dawn_xu_helloworld@163.com'
{"code":51,"error":true,"message":"There is no user named 'dawn_xu_helloworld@163.com'. Either you mis-typed the name or that user has not yet registered for a Gentoo's Bugzilla account.","documentation":"https://bugzilla.readthedocs.org/en/5.0/api/"}
```

Reply:
Your new email is still not registered. Can you login to Bugzilla with your new email address?
```
$ curl 'https://bugs.gentoo.org/rest/user?names=dawn_xu_helloworld@163.com'
{"code":51,"message":"There is no user named 'dawn_xu_helloworld@163.com'. Either you mis-typed the name or that user has not yet registered for a Gentoo's Bugzilla account.","error":true,"documentation":"https://bugzilla.readthedocs.org/en/5.0/api/"}
```